Archipelago.MultiClient.Net/ArchipelagoSessionFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Archipelago.MultiClient.Net.Cache;
+using Archipelago.MultiClient.Net.Helpers;
+
+namespace Archipelago.MultiClient.Net
+{
+    public static class ArchipelagoSessionFactory
+    {
+        public static ArchipelagoSession CreateSession(string hostUrl)
+        {
+            var socket = new ArchipelagoSocketHelper(hostUrl);
+            var dataPackageCache = new DataPackageFileSystemCache(socket);
+            
+            // Try to get datapackage just to send the GetDataPackage packet in case the cache is empty.
```

Comment:
The line is no longer valid and will be removed. At some point the function would send a GetDataPackage if the cache was empty but I have since removed extra GDP calls.